packages/react-query/src/v5.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-deprecated */
```

Comment:
```suggestion
```

Could you make @typescript-eslint/no-deprecated as warn in @suspensive/eslint-config?